src/Sentry.Unity.Android/SentryJava.cs

```diff
@@ -40,6 +42,55 @@ internal class SentryJava : ISentryJava
 {
     private static AndroidJavaObject GetSentryJava() => new AndroidJavaClass("io.sentry.Sentry");
 
+    public void Init(IJniExecutor jniExecutor, SentryUnityOptions options)
+    {
+        jniExecutor.Run(() =>
+        {
+            using var sentry = new AndroidJavaClass("io.sentry.android.core.SentryAndroid");
+            using var context = new AndroidJavaClass("com.unity3d.player.UnityPlayer")
+                .GetStatic<AndroidJavaObject>("currentActivity");
+
+            sentry.CallStatic("init", context, new OptionsConfiguration(androidOptions =>
```

Comment:
Can this throw? I imagine do this in a `try` `catch` and just degrade to C# only support is a good approach.
If it fails you can flip that flag that native support is off, and avoid the P/Invoke calls

Reply:
Yes. I've improved on this in https://github.com/getsentry/sentry-unity/pull/1927